arci-urdf-viz/src/client.rs

```diff
@@ -83,7 +83,8 @@ fn create_joint_trajectory_clients_inner(
             .into());
         }
         let client =
-            arci::PartialJointTrajectoryClient::new(config.joint_names, all_client.clone());
+            arci::PartialJointTrajectoryClient::new(config.joint_names, all_client.clone())
```

Comment:
Config.joint_names is 0 length.
What's happen?

Reply:
I think it should be an error.